omero2pandas/upload.py

```diff
@@ -131,11 +230,26 @@ def create_table(df, table_name, parent_id, parent_type, conn, chunk_size):
         link_obj.link(target_obj, annotation)
         link_obj = conn.getUpdateService().saveAndReturnObject(
             link_obj, _ctx={"omero.group": str(parent_group)})
-        LOGGER.info("Saved annotation link")
-
+        annotation_id = link_obj.child.id.val
+        LOGGER.info(f"Uploaded as FileAnnotation {annotation_id}")
+        extra_link_objs = []
+        unloaded_ann = omero.model.FileAnnotationI(annotation_id, False)
+        for ob_type, ob_id in link_to:
+            # Construct additional links
+            link_obj = LINK_TYPES[ob_type]()
+            target_obj = OBJECT_TYPES[ob_type](ob_id, False)
+            link_obj.link(target_obj, unloaded_ann)
+            extra_link_objs.append(link_obj)
+        if extra_link_objs:
+            try:
+                conn.getUpdateService().saveArray(
```

Comment:
I've seen this pattern in a few other places however It's absolutely not the intent to have the context passed in as a keyword argument. This really should be: `extra_link_objs, {"omero.group": str(parent_group)}`.